stable/aws-calico/README.md

```diff
@@ -1,3 +1,12 @@
+# Important Announcement: We have decided to stop maintaining and updating Calico charts in this repository #
+- The current Calico charts will not be updated
+- The current Calico charts will be removed from this repository on March 31, 2023
```

Comment:
Maybe we can merge the above two points into one..Something like below.

"Calico charts in this repository will no longer be updated and will be completely removed by March 31,2023"

Reply:
@achevuru I would like to make them to two points, which can keep them short and more noticeable. Please let me know you have concern. Thanks.

Reply:
Main reason I proposed to merge them was because "no longer update calico charts` portion of message is in the `Important Announcement` section and we're repeating the same in the first point. But that being said, I've no strong preferences either way..